src/compile/selection/interval.ts

```diff
@@ -118,14 +118,22 @@ const interval:SelectionCompiler = {
       name: undefined,
       type: 'rect',
       encode: {
-        enter: {fill: {value: '#eee'}},
+        enter: {
+          fill: {value: '#333'},
+          fillOpacity: {value: 0.125},
+          stroke: undefined
+        },
         update: update
       }
     }].concat(marks, {
       name: name + BRUSH,
       type: 'rect',
       encode: {
-        enter: {fill: {value: 'transparent'}},
+        enter: {
```

Comment:
This maybe a dumb question, but I wonder why we need two separate rects here for fill and stroke. (Maybe add a comment?)